scripts/powershell/sub-cleanup/exceptions.txt

```diff
@@ -0,0 +1,5 @@
+bterlson-testbed
```

Comment:
Why does `bterlson-testbed` need an exception?  Doesn't it already follow the guideline?

Reply:
His alias is brterlso